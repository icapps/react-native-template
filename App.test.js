import React from 'react';
import shallowTester from './src/utils/test/shallowTestHelper';
import App from './App';

shallowTester(<App />);
