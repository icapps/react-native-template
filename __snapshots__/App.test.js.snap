// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <Connect(Home) />
</Provider>
`;
